### DISCUSSION TITLE
Cannot open a specific Collabora document, but can make a copy of it and open that

### DISCUSSION THREAD

**Discussion by karypid** [nextcloud/all-in-one#4797] [Questions] [answered]
Hello,

Sometimes my Nextcloud AIO instance gets into this annoying state where it refuses to load a document. I searched for the error and found various posts, it basically comes up with:

>  Document loading failed
> Failed to establish socket connection or socket connection closed unexpectedly.

I've seena post about  [updating PHP](https://help.nextcloud.com/t/issue-after-upgrading-to-25-0-0/148609/11), which I assume is not my case, because it is a specific document that fails to load. If I make a copy of the same document in the same folder, the copy opens up fine.

This happens randomly and is annoying. On the logs of the "Collabora" container I see:

```
wsd-00007-00046 2024-06-08 17:43:32.099016 +0100 [ websrv_poll ] WRN  DocBroker [https%3A%2F%2Fnextcloud.mydomain.com%3A443%2Findex.php%2Fapps%2Frichdocuments%2Fwopi%2Ffiles%2F12693_ocigli027yu8] is unloading. Rejecting client request to load session [520]| wsd/ClientRequestDispatcher.cpp:139
wsd-00007-00046 2024-06-08 17:43:32.099042 +0100 [ websrv_poll ] ERR  #126: Failed to create DocBroker [https%3A%2F%2Fnextcloud.mydomain.com%3A443%2Findex.php%2Fapps%2Frichdocuments%2Fwopi%2Ffiles%2F12693_ocigli027yu8]: error: cmd=load kind=docunloading| wsd/RequestVettingStation.cpp:333
wsd-00007-00046 2024-06-08 17:43:32.112709 +0100 [ websrv_poll ] ERR  FileServerRequestHandler: File not found: Invalid URI request: [/browser/594b605/images/dark/lc_searchprev.svg].| wsd/FileServer.cpp:747
wsd-00007-00046 2024-06-08 17:43:32.115508 +0100 [ websrv_poll ] ERR  FileServerRequestHandler: File not found: Invalid URI request: [/browser/594b605/images/dark/lc_searchnext.svg].| wsd/FileServer.cpp:747
wsd-00007-00046 2024-06-08 17:43:32.139506 +0100 [ websrv_poll ] ERR  FileServerRequestHandler: File not found: Invalid URI request: [/browser/594b605/images/dark/lc_prev.svg].| wsd/FileServer.cpp:747
wsd-00007-00046 2024-06-08 17:43:32.141791 +0100 [ websrv_poll ] ERR  FileServerRequestHandler: File not found: Invalid URI request: [/browser/594b605/images/dark/lc_next.svg].| wsd/FileServer.cpp:747
wsd-00007-00046 2024-06-08 17:43:32.166393 +0100 [ websrv_poll ] ERR  FileServerRequestHandler: File not found: Invalid URI request: [/browser/594b605/images/dark/lc_zoom.svg].| wsd/FileServer.cpp:747
wsd-00007-00046 2024-06-08 17:43:32.221989 +0100 [ websrv_poll ] ERR  FileServerRequestHandler: File not found: Invalid URI request: [/browser/594b605/images/lc_zoom.svg].| wsd/FileServer.cpp:747
wsd-00007-00046 2024-06-08 17:43:32.901245 +0100 [ websrv_poll ] WRN  DocBroker [https%3A%2F%2Fnextcloud.mydomain.com%3A443%2Findex.php%2Fapps%2Frichdocuments%2Fwopi%2Ffiles%2F12693_ocigli027yu8] is unloading. Rejecting client request to load session [527]| wsd/ClientRequestDispatcher.cpp:139
wsd-00007-00046 2024-06-08 17:43:32.901271 +0100 [ websrv_poll ] ERR  #128: Failed to create DocBroker [https%3A%2F%2Fnextcloud.mydomain.com%3A443%2Findex.php%2Fapps%2Frichdocuments%2Fwopi%2Ffiles%2F12693_ocigli027yu8]: error: cmd=load kind=docunloading| wsd/RequestVettingStation.cpp:333
wsd-00007-00046 2024-06-08 17:43:35.175576 +0100 [ websrv_poll ] WRN  DocBroker [https%3A%2F%2Fnextcloud.mydomain.com%3A443%2Findex.php%2Fapps%2Frichdocuments%2Fwopi%2Ffiles%2F12693_ocigli027yu8] is unloading. Rejecting client request to load session [528]| wsd/ClientRequestDispatcher.cpp:139
wsd-00007-00046 2024-06-08 17:43:35.175602 +0100 [ websrv_poll ] ERR  #130: Failed to create DocBroker [https%3A%2F%2Fnextcloud.mydomain.com%3A443%2Findex.php%2Fapps%2Frichdocuments%2Fwopi%2Ffiles%2F12693_ocigli027yu8]: error: cmd=load kind=docunloading| wsd/RequestVettingStation.cpp:333
```

What does the above failure mean? I would like to get to the bottom of this. I think waiting a while or restarting Nextcloud AIO resolvers this, but I would really want to find out and fix the root cause.

**Comment by karypid** [comment]
I just restarted the nextcloud-aio-collabora container and the document now opens fine. So something seems to get "stuck" in that container that goes away with a restart. Would love to get some guidance on what to look for when this happens.

**Comment by szaimen** [comment, marked as answer]
> ```
> wsd-00007-00046 2024-06-08 17:43:32.099016 +0100 [ websrv_poll ] WRN  DocBroker [https%3A%2F%2Fnextcloud.mydomain.com%3A443%2Findex.php%2Fapps%2Frichdocuments%2Fwopi%2Ffiles%2F12693_ocigli027yu8] is unloading. Rejecting client request to load session [520]| wsd/ClientRequestDispatcher.cpp:139
> wsd-00007-00046 2024-06-08 17:43:32.099042 +0100 [ websrv_poll ] ERR  #126: Failed to create DocBroker [https%3A%2F%2Fnextcloud.mydomain.com%3A443%2Findex.php%2Fapps%2Frichdocuments%2Fwopi%2Ffiles%2F12693_ocigli027yu8]: error: cmd=load kind=docunloading| wsd/RequestVettingStation.cpp:333
> wsd-00007-00046 2024-06-08 17:43:32.112709 +0100 [ websrv_poll ] ERR  FileServerRequestHandler: File not found: Invalid URI request: [/browser/594b605/images/dark/lc_searchprev.svg].| wsd/FileServer.cpp:747
> wsd-00007-00046 2024-06-08 17:43:32.115508 +0100 [ websrv_poll ] ERR  FileServerRequestHandler: File not found: Invalid URI request: [/browser/594b605/images/dark/lc_searchnext.svg].| wsd/FileServer.cpp:747
> wsd-00007-00046 2024-06-08 17:43:32.139506 +0100 [ websrv_poll ] ERR  FileServerRequestHandler: File not found: Invalid URI request: [/browser/594b605/images/dark/lc_prev.svg].| wsd/FileServer.cpp:747
> wsd-00007-00046 2024-06-08 17:43:32.141791 +0100 [ websrv_poll ] ERR  FileServerRequestHandler: File not found: Invalid URI request: [/browser/594b605/images/dark/lc_next.svg].| wsd/FileServer.cpp:747
> wsd-00007-00046 2024-06-08 17:43:32.166393 +0100 [ websrv_poll ] ERR  FileServerRequestHandler: File not found: Invalid URI request: [/browser/594b605/images/dark/lc_zoom.svg].| wsd/FileServer.cpp:747
> wsd-00007-00046 2024-06-08 17:43:32.221989 +0100 [ websrv_poll ] ERR  FileServerRequestHandler: File not found: Invalid URI request: [/browser/594b605/images/lc_zoom.svg].| wsd/FileServer.cpp:747
> wsd-00007-00046 2024-06-08 17:43:32.901245 +0100 [ websrv_poll ] WRN  DocBroker [https%3A%2F%2Fnextcloud.mydomain.com%3A443%2Findex.php%2Fapps%2Frichdocuments%2Fwopi%2Ffiles%2F12693_ocigli027yu8] is unloading. Rejecting client request to load session [527]| wsd/ClientRequestDispatcher.cpp:139
> wsd-00007-00046 2024-06-08 17:43:32.901271 +0100 [ websrv_poll ] ERR  #128: Failed to create DocBroker [https%3A%2F%2Fnextcloud.mydomain.com%3A443%2Findex.php%2Fapps%2Frichdocuments%2Fwopi%2Ffiles%2F12693_ocigli027yu8]: error: cmd=load kind=docunloading| wsd/RequestVettingStation.cpp:333
> wsd-00007-00046 2024-06-08 17:43:35.175576 +0100 [ websrv_poll ] WRN  DocBroker [https%3A%2F%2Fnextcloud.mydomain.com%3A443%2Findex.php%2Fapps%2Frichdocuments%2Fwopi%2Ffiles%2F12693_ocigli027yu8] is unloading. Rejecting client request to load session [528]| wsd/ClientRequestDispatcher.cpp:139
> wsd-00007-00046 2024-06-08 17:43:35.175602 +0100 [ websrv_poll ] ERR  #130: Failed to create DocBroker [https%3A%2F%2Fnextcloud.mydomain.com%3A443%2Findex.php%2Fapps%2Frichdocuments%2Fwopi%2Ffiles%2F12693_ocigli027yu8]: error: cmd=load kind=docunloading| wsd/RequestVettingStation.cpp:333
> ```

Hi, this sounds like a bug in https://github.com/CollaboraOnline/online/issues or https://github.com/nextcloud/richdocuments/issues. Please report this there
